sql/catalyst/src/main/java/org/apache/spark/sql/connector/expressions/Expressions.java

```diff
@@ -46,9 +47,9 @@ public static Transform apply(String name, Expression... args) {
   }
 
   /**
-   * Create a named reference expression for a column.
+   * Create a named reference expression for a (nested) column.
    *
-   * @param name a column name
+   * @param name The column name. It refers to nested column if name contains dot.
```

Comment:
This isn't true. Names are parsed using the SQL parser, so escaped names are valid.